src/event-topics/event-subscription-topic-accountingintegration.markdown

```diff
@@ -10,41 +10,170 @@ layout: reference
+> **Limitation**: This event subscription is only available to partners who have been granted access to Accounting Integration API.
 
 ## <a name="scope-usage"></a>Scope Usage
 
-Name|Description
----|---
-`spend.accountingintegration.read`|Accounting Integration v4 API related events.
+|Name|Description|
+|---|---|
+|`spend.accountingintegration.read`|Accounting Integration API related events|
 
 ## <a name="events"></a>Events
 
-Name|Condition|User Case
----|---|---
-`listMapping.created`|This event will be published after the customer has mapped the SAP Concur systems list with the ERP list name.|The subscriber may send over the list items.
-`listMapping.deleted`|This event will be published after the customer has removed the mapping between the SAP Concur systems list and the ERP list name.|The subscriber may remove the list items from SAP Concur.
-`data.resetRequested`|This event will be published when the customer has requested to reset the ERP data cache.|The subscriber may reset the ERP data cache.
-`data.statusUpdated`|This event will be published after the completion of the processing of the ERP data.|The subscriber may query for the status details.
+|Event Type|Condition that triggers event|Recommended Partner Action|
+|---|---|---|
+|`listMapping.created`| This event will be published after the customer administrator has mapped the SAP Concur list with the ERP list name. (Professional Edition Note: The customer admin must first import the list, then map them in the `Forms and Fields` interface. Multiple fields can be linked to a single list. Only the first mapping of a list will trigger this event) | This is a trigger telling the Partner that it's time for the ERP data to be synchronized (Via the List Items API) with SAP Concur data.|
+|`listMapping.deleted`| This event will be published after the customer administrator has removed the mapping between the SAP Concur list and the ERP list name. (Professional Edition Note: As multiple-fields can be mapped to the same list - this event will only trigger when all fields are unmapped from a given list) | The indicated list is no longer mapped and allows the Partner to do it's own data cleanup. This should also stop the indicated list data from being synchronized.|
+|`data.resetRequested`| This event will be published when the customer administrator has requested to reset the ERP data cached in SAP Concur data-center. | Data should be resynchronized to SAP Concur systems.|
+|`data.statusUpdated`| This event will be sent to notify the partner that the data they sent to SAP Concur systems via the PATCH action with the `Accounts`, `Lists` and `Vendors` APIs has been processed on SAP Concur side. One notification is sent for each PATCH request that the partner performs. | The status may be interesting for the Partner to perform actions such as notification, logging, cleanup, etc.|
```

Comment:
"side" isn't an approved trademark descriptor. Can you just say "our side" and drop SAP Concur. They're reading our content so that should be clear.

Reply:
This change has been made.